README.md

```diff
@@ -1,2 +1,56 @@
 # publisher-frontend
 Publisher frontend to manage course creation and marketing content curation.
+
+# Important Note
+
+This repository is in early Alpha stages of development and will be for the foreseeable future. Until then it is 
+not recommended for use in production.
+
+# Development Environment
+
+## Getting Started
+
+Run the following commands to get started with the Publisher Development environment.
+
+1. Install Docker locally. 
+    * Docker is used to generate a local development environment that contains all the required tools and libraries to 
+    develop against. 
+    
+    * The most recent stable version of Docker should work.
+    
+2. Run the following docker-compose commands to get started.
+
+    ```
+    docker-compose build
+    docker-compose up 
+    ```
+    
+    At the end of these two commands running the web application should be running. You should be able to view it 
+    in a Web Browser at `localhost:18400`.
```

Comment:
How do we pick these port numbers btw? Do we have our own little internal ICANN agency? :)

Reply:
I don't think we do, I just picked a port /shrug.

Are you concerned about this port locally?

Reply:
No, just musing.